assets/src/scss/layout/_page-header.scss

```diff
@@ -126,26 +116,12 @@
   font-family: $roboto;
   font-weight: bold;
   text-transform: none;
-  max-width: 80%;
-  margin-bottom: 10px;
-
```

Comment:
Do we intend to also change this for pages and campaign pages? It could also cause issues with background image alignment. Maybe we should still include the removed rules for pages and campaign pages in a separate file? :thinking: That would be easier to do if we had code splitting per post type, so we wouldn't have to scope the CSS.

Reply:
True! I've just pushed a commit to move these changes to `_post.scss` instead so it won't affect pages and campaigns 👍